### ISSUE TITLE
Ideas of things to implement on Upsilon and errors

### ISSUE THREAD

**Issue by theodorechle** [UpsilonNumworks/Upsilon#298]
- [ ] a file manager and the possibility to give any extension you want for a file
- [ ] a way to add files on the reader app directly on the calculator
- [ ] formulas like for volumes of geometrical forms
- [ ]  in python app, you can't see all the parameters to enter in the draw_string function in kandinsky and the fill_polygon function is automatically created with 4 parameters to enter but there is only two
- [x] add the possibility to write in italic with the draw_string function in python
- [ ] add a character table
- [ ] A way to get docstrings in python and inputs/outputs of functions (.__doc__, help)
- [ ] The possibility to change the color of the displayed str with a (r,g,b) argument (and hexadecimal str) for prints (maybe a new special print function)

**Comment by lolocomotive** [comment]
3 and 5 are interesting. I think 5 shouldn't be too hard to implement

**Comment by theodorechle** [comment]
Cool !

**Comment by Overengined** [comment]
3 would collide with exam protocols in France I think, so better to have the formulas removed in exam mode.

**Comment by theodorechle** [comment]
Yes, I forget that but I think since they are implemented this will be not really hard to block them.

**Comment by NowanPython** [comment]
I like all the ideas :+1:
